back-end/main.py

```diff
@@ -43,7 +43,7 @@ async def get_platforms():
     return platforms
 
 
-@app.get("/senders", response_model=Dict[Platform, List[Sender]])
+@app.get("/senders", response_model=Dict[int, List[Sender]])
```

Comment:
[Should]
Không liên quan nhưng nếu em theo rest (90% mn project đều theo rest btw =))) thì các endpoint khác nên dùng danh từ số nhiều nha.
Anh thấy trong code có create-templates với send-message.
Nếu em làm cả BE và FE thì không sao nhưng nếu có thêm người khác thì sẽ không biết được tên của send-message vì họ sẽ expect nó là create-messages.
Nên viết các endpoint có cú pháp giống nhau để dễ tìm kiếm hơn:
/messages và /templates